content/en/resources/asynchronous-data-science-with-pyensign.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Asynchronous Data Science With Pyensign"
+slug: "asynchronous-data-science-with-pyensign"
+draft: false
+event_date: "2023-06-20"
+image: "img/resources/webinars/asynchronous-data-science.webp"
+name: "Asynchronous Data Science with PyEnsign"
+description: |
```

Comment:
`|` was added because the content spans multiple lines. This allow the markdown to be rendered properly.